integ/README.md

```diff
@@ -3,22 +3,43 @@
 To run all test suites:
 
 1. Build and install dependencies by running build.sh from the top-level RFDK directory
+1. Configure AWS credentials (tests will use the default AWS profile, so either set up a default profile in `~/.aws/`
```

Comment:
Instance role would be used as well, no? So there's a third option to temp creds or locally-stored creds

Reply:
Yes, I can elaborate on this to list and link to the suggested methods as documented in the [JS SDK Dev Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials-node.html)